src/storage/src/hummock/sstable/builder.rs

```diff
@@ -299,7 +299,7 @@ impl<W: SstableWriter, F: FilterBuilder> SstableBuilder<W, F> {
                 "A large key (table_id={}, len={}, epoch={}) is added to block",
                 table_id,
                 table_key_len,
-                full_key.epoch
+                full_key.epoch_with_gap.as_u64()
```

Comment:
We can impl Debug for EpochWithGap as printing its pure_epoch and offset separately.